### ISSUE TITLE
Open Releases

### ISSUE THREAD

**Issue by iPaulPro** [CloutFeed/mobileApp#119]
Is there a plan to openly build releases, using a Continuous Delivery system like GitHub Actions?

Right now there is no way to verify what code actually makes it into a binary, and there are no tagged releases to even have a frame of reference. We've seen updates on the Google Play store that are clearly ahead of this repo, which means there are essentially 2 code sources.

**Comment by RHatem** [comment]
Hi Paul, 
Sure this is on the road map. Could you please give me more details about the case you are talking about? Because I always push the code to the repository before the deployment and this is the only code source we are using for the project.
You can also track the development of new features live on this repo.

**Comment by iPaulPro** [comment]
A couple people reported to me that your Android release on July 4th had features that were not yet in the repo.

The problem is that we just have to take your word that this is the code base that is compiled. It's really not a very secure setup for users.

**Comment by RHatem** [comment]
Yes, I absolutely understand the problem you are mentioning and will try to prioritize GitHub actions.

All the features that were contained in this release were pushed max on the 4th of July. Last commit was the CloutCast integration on the same day. #76 

The only thing that sometimes is not included in the repo is the version increase but will change that in the future and add release tags like you suggested. Need more details about the "missing features code" to understand if there was really some kind of mistake and try to avoid that in the future.

**Comment by AdityaChaudhary0005** [comment]
So first of all let me clarify this. This was started with me. One day i thought to make some PR in CloutFeed so i searched "CloutFeed" on github and the only repo i could find in the search result was this:  https://github.com/1R053/mobileApp .
You can try it yourself  ¯\_(ツ)_/¯
You can almost never discover this repo if you search cloutfeed on github
I was mistaken that that was the repo for cloutfeed app and after seeing that the repo was never updated (while the app was) i reported it to @iPaulPro .

I really apologise for this misunderstanding.
